### ISSUE TITLE
Unknown provider: $$isDocumentHiddenProvider

### ISSUE THREAD

**Issue by deywibkiss** [jadjoubran/laravel5-angular-material-starter#460]
**support request**
Browser console shows me the following error:

`final-fb558dc2f3.js:68 Uncaught Error: [$injector:unpr] Unknown provider: $$isDocumentHiddenProvider <- $$isDocumentHidden <- $$animateQueue <- $animate <- $compile <- $$animateQueue`

And I don't know where to make the fix. Any help?

* **Laravel & Angular version:** 3.4.0

**Comment by flick36** [comment]
can you show your `bower.json` please?, it's a problem with `angular-animate`

**Comment by DennisOng** [comment]
Yup ran into this too yesterday, in my case reverting to these versions of angular-animate and aria solved it

 "dependencies": {
    "angular-animate": "1.5.9",
    "angular-aria": "1.5.9",
}

**Comment by deywibkiss** [comment]
Thanks for answering so soon. This is my `bower.json`

```
{
  "name": "laravel5-angular-material-starter",
  "authors": [
    "Jad Joubran <jad@jadjoubran.com>"
  ],
  "ignore": [
    "**/.*",
    "node_modules",
    "bower_components",
    "test",
    "tests"
  ],
  "dependencies": {
    "angular": "~1.5",
    "angular-ui-router": "~1.0.0-alpha.3",
    "ngstorage": "~0.3.10",
    "angular-material": "~1.1.1",
    "angular-animate": "1.5.9",
    "angular-aria": "1.5.9",
    "restangular": "~1.5.2",
    "angular-loading-bar": "~0.9.0",
    "satellizer": "^0.14.0"
  }
}

```

Then I ran `bower install`  and`gulp && gulp watch` but nothing happens. There's still the error.

**Comment by flick36** [comment]
remove `angular-animate`, and `angular-aria`, those are added automatically by `angular-material`

**Comment by deywibkiss** [comment]
I had to remove `angular-animate` and `angular-aria` as @flick36 said, then I run the command:
`bower install "angular-animate"` so bower answered me again which version of angular I'd wanted to install and I chose one compatible with `angular-animate 1.5.9`. Thanks.

**Comment by flick36** [comment]
Yes, you have to keep consistency between versions, normally if you are using `angular` version 1.5.3 for example, you should use the same version for `angular-{something}` in this case... if you are using `angular 1.5.9` you should use `angular-aria` and `angular-animate` `1.5.9`

**Comment by nueko** [comment]
confirm the problem with angular > 1.5.9

**Comment by deywibkiss** [comment]
@nueko as @flick36 said, you need to keep the consistency between versions. Please verify that your angular and angular plugins all of them have the same version.

**Comment by ArrayZoneYour** [comment]
When you first run `bower install` and you will see that
```
1) angular#~1.5 which resolved to 1.5.11 and is required by laravel5-angular-material-starter
2) angular#1.6.1 which resolved to 1.6.1 and is required by angular-animate#1.6.1, angular-aria#1.6.1, angular-messages#1.6.1
```
It means that angular-animate#1.6.1, angular-aria#1.6.1, angular-messages#1.6.1 need angular#1.6.1  but laravel5-angular-material-starter need angular 1.5.~,you can choose to reverte three package to 1.5.9, but use angular1.6.1 also works.
